src/command-line/config.js

```diff
@@ -6,9 +6,12 @@ program
 	.command("config")
 	.description("Edit config: " + Helper.CONFIG_PATH)
 	.action(function() {
-		child.spawn(
+		var child_spawn = child.spawn(
 			process.env.EDITOR || "vi",
 			[Helper.CONFIG_PATH],
 			{stdio: "inherit"}
 		);
+		child_spawn.on("error", function() {
+			log.error("$EDITOR is not set, and vi was not found.");
```

Comment:
What I would do is print `Helper.CONFIG_PATH` here so that users can easily edit the file in their favourite editor.